iaen.edu.ec,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
iainbengkulu.ac.id,"HTTPSConnectionPool(host='iainbengkulu.ac.id', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='iainbengkulu.ac.id', port=443): Read timed out. (read timeout=2)"
iainkediri.ac.id,"HTTPSConnectionPool(host='iainkediri.ac.id', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='iainkediri.ac.id', port=80): Read timed out. (read timeout=2)"
iainponorogo.ac.id,"HTTPSConnectionPool(host='iainponorogo.ac.id', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f31992ed070>: Failed to establish a new connection: [Errno 113] No route to host'))","HTTPConnectionPool(host='iainponorogo.ac.id', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f3197f561c0>, 'Connection to iainponorogo.ac.id timed out. (connect timeout=2)'))"
iam.ma,"HTTPSConnectionPool(host='iam.ma', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.iam.ma', port=443): Max retries exceeded with url: /Pages/index.aspx (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
ias.co.il,"HTTPSConnectionPool(host='ias.co.il', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='ias.co.il', port=80): Read timed out. (read timeout=2)"
ibict.br,"HTTPSConnectionPool(host='ibict.br', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='ibict.br', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
ibtimes.com,"HTTPSConnectionPool(host='www.ibtimes.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.ibtimes.com', port=443): Read timed out. (read timeout=2)"
icbc.com.ar,"HTTPSConnectionPool(host='icbc.com.ar', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))","HTTPSConnectionPool(host='icbc.com.ar', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))"
icbf.gov.co,"HTTPSConnectionPool(host='icbf.gov.co', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f3195c1a820>, 'Connection to icbf.gov.co timed out. (connect timeout=2)'))","HTTPConnectionPool(host='icbf.gov.co', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f319679dee0>, 'Connection to icbf.gov.co timed out. (connect timeout=2)'))"
icm.edu.pl,"HTTPSConnectionPool(host='icm.edu.pl', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='icm.edu.pl', port=443): Read timed out. (read timeout=2)"
idi.org.il,"HTTPSConnectionPool(host='idi.org.il', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f31969503a0>, 'Connection to idi.org.il timed out. (connect timeout=2)'))","HTTPConnectionPool(host='idi.org.il', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3195970040>: Failed to establish a new connection: [Errno 113] No route to host'))"
ieatishootipost.sg,"HTTPConnectionPool(host='ieatishootipost.sg', port=80): Read timed out. (read timeout=2)","HTTPConnectionPool(host='ieatishootipost.sg', port=80): Read timed out. (read timeout=2)"
iedu.sk,"HTTPSConnectionPool(host='iedu.sk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))","HTTPSConnectionPool(host='iedu.sk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
iefp.pt,"HTTPSConnectionPool(host='iefp.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f31943ca490>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='iefp.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3194ca9d90>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ieftine.eu,Exceeded 30 redirects.,Exceeded 30 redirects.
iett.istanbul,"HTTPSConnectionPool(host='iett.istanbul', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='iett.istanbul', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
igm.org.il,"HTTPSConnectionPool(host='igm.org.il', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='igm.org.il', port=80): Read timed out. (read timeout=2)"
iho.hu,"HTTPSConnectionPool(host='iho.hu', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='iho.hu', port=443): Read timed out. (read timeout=2)"
iif.hu,"HTTPSConnectionPool(host='iif.hu', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f31934d9af0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='iif.hu', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3194a44f70>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ijf.hr,"HTTPSConnectionPool(host='ijf.hr', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f3196ab6f10>, 'Connection to ijf.hr timed out. (connect timeout=2)'))","HTTPConnectionPool(host='ijf.hr', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f3193ab6df0>, 'Connection to ijf.hr timed out. (connect timeout=2)'))"
ijunoon.com,"HTTPSConnectionPool(host='ijunoon.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='ijunoon.com', port=443): Read timed out. (read timeout=2)"
ilgiornale.it,"HTTPSConnectionPool(host='www.ilgiornale.it', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.ilgiornale.it', port=443): Read timed out. (read timeout=2)"
ilmangione.it,"HTTPSConnectionPool(host='ilmangione.it', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.ilmangione.it', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
ilmusosial.id,"HTTPSConnectionPool(host='ilmusosial.id', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f31969f9790>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ilmusosial.id', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3191f44a90>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ilpapirocartolibreria.it,"HTTPSConnectionPool(host='ilpapirocartolibreria.it', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='ilpapirocartolibreria.it', port=80): Read timed out. (read timeout=2)"
imamu.edu.sa,"HTTPSConnectionPool(host='imamu.edu.sa', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='imamu.edu.sa', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
imge.com.tr,"HTTPSConnectionPool(host='www.imge.com.tr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.imge.com.tr', port=443): Read timed out. (read timeout=2)"
imged.pl,"HTTPSConnectionPool(host='imged.pl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f31966f7be0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='imged.pl', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3191d2fcd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
imist.ma,"HTTPSConnectionPool(host='imist.ma', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f319669c640>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='imist.ma', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f31920fb790>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
immoproxio.be,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
imommy.gr,"HTTPSConnectionPool(host='imommy.gr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.imommy.gr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))"
imprentanacional.go.cr,"HTTPSConnectionPool(host='imprentanacional.go.cr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f31986a7c40>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='imprentanacional.go.cr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f31978c8a60>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
impreseita.com,"HTTPSConnectionPool(host='impreseita.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3197904640>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='impreseita.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f31970b5670>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
imt-ip.pt,"HTTPSConnectionPool(host='imt-ip.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3196dc4c10>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='imt-ip.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f319687b760>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
imtranslator.net,"HTTPSConnectionPool(host='imtranslator.net', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='imtranslator.net', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
imunita.online,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
in.gr,"HTTPSConnectionPool(host='in.gr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.in.gr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))"
inafed.gob.mx,"HTTPSConnectionPool(host='inafed.gob.mx', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f318f0f4fd0>, 'Connection to inafed.gob.mx timed out. (connect timeout=2)'))","HTTPConnectionPool(host='inafed.gob.mx', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f3197572700>, 'Connection to inafed.gob.mx timed out. (connect timeout=2)'))"
inclusion.gob.ec,"HTTPSConnectionPool(host='inclusion.gob.ec', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'inclusion.gob.ec' doesn't match either of 'www.agricultura.gob.ec', 'agricultura.gob.ec'"")))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
indap.gob.cl,"('Connection aborted.', BadStatusLine('\n'))","HTTPConnectionPool(host='www.indap.gob.cl', port=80): Read timed out. (read timeout=2)"
indexlivingmall.com,"HTTPSConnectionPool(host='indexlivingmall.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f318eeebf40>, 'Connection to indexlivingmall.com timed out. (connect timeout=2)'))","HTTPConnectionPool(host='indexlivingmall.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f318f2c7970>, 'Connection to indexlivingmall.com timed out. (connect timeout=2)'))"
indiacom.com,"HTTPSConnectionPool(host='indiacom.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f318eecf910>, 'Connection to indiacom.com timed out. (connect timeout=2)'))","HTTPConnectionPool(host='indiacom.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f3191f54610>, 'Connection to indiacom.com timed out. (connect timeout=2)'))"
indonesia.go.id,"HTTPSConnectionPool(host='indonesia.go.id', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.indonesia.go.id', port=443): Read timed out. (read timeout=2)"
ine.gub.uy,"HTTPSConnectionPool(host='ine.gub.uy', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='ine.gub.uy', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
inefop.org.uy,"HTTPSConnectionPool(host='inefop.org.uy', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3196673ee0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='inefop.org.uy', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f319600ae50>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
inews.bg,"HTTPSConnectionPool(host='inews.bg', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='inews.bg', port=443): Read timed out. (read timeout=2)"
infarmed.pt,"HTTPSConnectionPool(host='infarmed.pt', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f31982d8fa0>, 'Connection to infarmed.pt timed out. (connect timeout=2)'))","HTTPConnectionPool(host='infarmed.pt', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f31974203d0>, 'Connection to infarmed.pt timed out. (connect timeout=2)'))"
influenster.com,"HTTPSConnectionPool(host='www.influenster.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.influenster.com', port=443): Read timed out. (read timeout=2)"
infocenters.co.il,"HTTPSConnectionPool(host='infocenters.co.il', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='infocenters.co.il', port=443): Read timed out. (read timeout=2)"
infogate.cl,"HTTPSConnectionPool(host='www.infogate.cl', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.infogate.cl', port=443): Read timed out. (read timeout=2)"
infogol.net,"HTTPSConnectionPool(host='infogol.net', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.infogol.net', port=443): Read timed out. (read timeout=2)"
infoguia.co.cr,"HTTPSConnectionPool(host='infoguia.co.cr', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f318eac4a30>, 'Connection to infoguia.co.cr timed out. (connect timeout=2)'))","HTTPConnectionPool(host='infoguia.co.cr', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f318eb2f7c0>, 'Connection to infoguia.co.cr timed out. (connect timeout=2)'))"
infoguia.com,"HTTPSConnectionPool(host='infoguia.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f318eb2fdf0>, 'Connection to infoguia.com timed out. (connect timeout=2)'))","HTTPConnectionPool(host='infoguia.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f318e9c62b0>, 'Connection to infoguia.com timed out. (connect timeout=2)'))"
infojuht.ee,"HTTPSConnectionPool(host='infojuht.ee', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='infojuht.ee', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
infomed.co.il,"HTTPSConnectionPool(host='infomed.co.il', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f318f2bed90>, 'Connection to infomed.co.il timed out. (connect timeout=2)'))","HTTPConnectionPool(host='infomed.co.il', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f318ed3ef40>, 'Connection to infomed.co.il timed out. (connect timeout=2)'))"
infonegocios.biz,"HTTPSConnectionPool(host='infonegocios.biz', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='infonegocios.biz', port=443): Read timed out. (read timeout=2)"
infonegocios.info,"HTTPSConnectionPool(host='infonegocios.info', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='infonegocios.info', port=443): Read timed out. (read timeout=2)"
infoprision.com,"HTTPSConnectionPool(host='infoprision.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'infoprision.com' doesn't match 'www.infoprision.com'"")))","HTTPSConnectionPool(host='infoprision.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'infoprision.com' doesn't match 'www.infoprision.com'"")))"
infoproff.com,"HTTPSConnectionPool(host='infoproff.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.infoproff.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
infopublik.id,"HTTPSConnectionPool(host='infopublik.id', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='infopublik.id', port=443): Read timed out. (read timeout=2)"
informa.es,"HTTPSConnectionPool(host='www.informa.es', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.informa.es', port=443): Read timed out. (read timeout=2)"
information.dk,"HTTPSConnectionPool(host='information.dk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.information.dk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))"
ing.be,"HTTPSConnectionPool(host='ing.be', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='ing.be', port=80): Read timed out. (read timeout=2)"
inia.uy,"HTTPSConnectionPool(host='inia.uy', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f3197841c40>, 'Connection to inia.uy timed out. (connect timeout=2)'))","HTTPConnectionPool(host='inia.uy', port=80): Read timed out. (read timeout=2)"
ins-cr.com,"HTTPSConnectionPool(host='ins-cr.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3194ca93d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ins-cr.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3190011d30>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
intelec.co.cr,"HTTPSConnectionPool(host='intelec.co.cr', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='intelec.co.cr', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f318e5d4f70>, 'Connection to intelec.co.cr timed out. (connect timeout=2)'))"
internimedicina.cz,"HTTPSConnectionPool(host='internimedicina.cz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))","HTTPSConnectionPool(host='internimedicina.cz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))"
intima.hu,Exceeded 30 redirects.,Exceeded 30 redirects.
investor.bg,"HTTPSConnectionPool(host='www.investor.bg', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.investor.bg', port=443): Read timed out. (read timeout=2)"
invima.gov.co,"HTTPSConnectionPool(host='invima.gov.co', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='invima.gov.co', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
io.ua,"HTTPSConnectionPool(host='io.ua', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f318f0b4e20>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPConnectionPool(host='io.ua', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f318f32efd0>: Failed to establish a new connection: [Errno 111] Connection refused'))"
ioc.ee,"HTTPSConnectionPool(host='ioc.ee', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.ioc.ee', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
ipapercms.dk,"HTTPSConnectionPool(host='ipapercms.dk', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f319777f550>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ipapercms.dk', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3197868e80>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ipleaders.in,"HTTPSConnectionPool(host='ipleaders.in', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='ipleaders.in', port=443): Read timed out. (read timeout=2)"
ipleiria.pt,"HTTPSConnectionPool(host='ipleiria.pt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='ipleiria.pt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
ipoor.org,"HTTPSConnectionPool(host='ipoor.org', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3196c70250>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ipoor.org', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f318f2bebb0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ipsantarem.pt,"HTTPSConnectionPool(host='ipsantarem.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f318eb36a00>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ipsantarem.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3191fa0580>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ipst.ac.th,"HTTPSConnectionPool(host='ipst.ac.th', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f318e9d0a60>, 'Connection to ipst.ac.th timed out. (connect timeout=2)'))","HTTPConnectionPool(host='ipst.ac.th', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f318f7ff610>, 'Connection to ipst.ac.th timed out. (connect timeout=2)'))"
ipvc.pt,"HTTPSConnectionPool(host='ipvc.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f318e8964c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ipvc.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f318e840c70>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ir.lv,"HTTPSConnectionPool(host='ir.lv', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='ir.lv', port=443): Read timed out. (read timeout=2)"
ird.gov.lk,"HTTPSConnectionPool(host='ird.gov.lk', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f318fb36430>: Failed to establish a new connection: [Errno 113] No route to host'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
